### PR TITLE
VSCode Interactive Instructions

### PR DESCRIPTION
**Pull request by tzaffi**
Proposing a change to the README which explains how to get an interactive debugger going using the Sandbox's indexer.

Also applying [markdownlint](https://github.com/DavidAnson/markdownlint)

Also, Not sure if this is the ideal place. It could exist as a separate README, or even on the dev portal.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -222,18 +240,64 @@ Algorand smart contract debugging process uses `tealdbg` command line of algod i
...
+`~$ ./sandbox tealdbg debug ${TEAL_PROGRAM} -f web -d dryrun.json`
+
 The debugging endpoint port (default 9392) is forwarded directly to the host machine and can be used directly by Chrome Dev Tools for debugging Algorand TEAL smart comtracts (Goto url chrome://inspect/ and configure port 9392 before using please).
 
 Note: If a different port is needed than the default, it may be changed by running `tealdbg --port YOUR_PORT` then modifying the docker-compose.yml file and change all occurances of mapped 9392 port with the desired one.
 
-## Errors
+## Sandbox Interactive Debugging with VSCode's `Remote - Container` Extension
+
+Here are steps you can take if you want to run an interactive debugger with an `indexer` running on the sandbox. Analogous instructions work for `algod` as well.
```
Comment (barnjamin) [inline README.md:257]: Can you please mention _why_ someone would want to do this and that its probably a more advanced thing to do

Comment (tzaffi) [reply README.md:257]: 👍

**Review by barnjamin — APPROVED** [review]
Thank you!

**Review by winder — APPROVED** [review]
Cool!

**Pull request merged**